src/server/utils/jsDependencies.js

```diff
@@ -0,0 +1,39 @@
+const babylon = require('@babel/parser');
```

Comment:
moving babel parser in utils as suggested in comments to be reused for `typescript`

Reply:
let's duplicate. It's one of that cases when better not DRY. It's better to keep all languages completely independent

Reply:
as said above, fine to duplicate the config but I suggest to keep the `getImports` in utils providing the config as params, what do you think?

Reply:
alright, sounds ok